<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_mouse_label" name="lbl_mouse">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Mouse area</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mouse_pos">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>X=0  Y=0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wdg_plot_x" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnstart_main_timer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start [main timer]</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop_main_timer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop [main timer]</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wdg_plot_y" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_mouse_vel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Vx=0  Vy=0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wdg_plot_vel" native="true">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_position_sensor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Position sensor ON</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbx_trajectories">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>161</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Trajectories</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rbt_trajectory_mouse">
       <property name="text">
        <string>Mouse</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbt_trajectory_sine">
       <property name="text">
        <string>Sine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbt_trajectory_cvel">
       <property name="text">
        <string>C. Velocity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbt_trajectory_cacc">
       <property name="text">
        <string>C. Acceleration</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btn_clear_image">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear stuff</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_noise">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add some noise</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>my_mouse_label</class>
   <extends>QLabel</extends>
   <header location="global">my_mouse_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
